nextclade/config/config.yaml

```diff
@@ -8,8 +8,8 @@ clade-iib:
 clade-i:
   root: "reconstructed_ancestral_mpox"
   min_date: ""
-  exclude_where: "--exclude-where clade!=I"
-  subsampling: "--subsample-max-sequences 300 --group-by clade lineage year month country --probabilistic-sampling"
+  exclude_where: "--exclude-where clade=II clade=IIa clade=IIb clade=outgroup"
```

Comment:
Can this be inverted to match #284?

https://github.com/nextstrain/mpox/blob/831138beb22ee157a410b65204c1c1e1599eb018/phylogenetic/defaults/clade-i/config.yaml#L29